.eslintrc

```diff
@@ -1,16 +1,25 @@
 {
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
```

Comment:
I thought that if you use the `plugin:xxx/recommended` you don't need to add them in `plugins`.

Reply:
And you thought right. Thins is I tried like you said before, and for some reason it didn't work, but I've working now.

Reply:
Super!